### ISSUE TITLE
shield tv issue - passthrough not working anymore

### ISSUE THREAD

**Issue by Rohazon** [koying/SPMC#859]
Hello all,

I have a 2015 shield and has been using spmc since begining 2016, passthrough was (nearly) always working with any format (HD or not) and was really enjoying it. And I believe since major nvidia update (a month or 2) I Have lost it all.
I did follow last spmc recomandation for passthrough in wiki
I was using 1.4 hdmi cable before so i switch to  2.0 hdmi cable (as it said that 1.4 can bring trouble)
but i still can't get any format to be passthrough.
I've tried various settings, previous build but nothing worked
I'm using a pioneer vsx s510

did i miss something or will the next build will fix it?

**Comment by edwoodjr** [comment]
I have the exact same issue but It just started from the last spmc update.

**Comment by BigOkie** [comment]
If I use any setting on the pass through options sometimes I will have issues getting sound to play. Only solution I've found is to cycle the sound input or do something to change it. I've gone back to using raw/raw in my settings as sometimes using iec causes sync issues. I have an older Onkyo AVR. Not too much of hassle to get all modes of sound output.

**Comment by grammetry** [comment]
Same issue here.
Since my Shield TV 2015 update to version 5.x
The audio-passthrough never work.
Just like you that I try anything but no work at all.
My whole stereo system become useless.
There are so many complain on Shiled TV community.
I guess the only way is waiting the new update coming.

**Comment by Rohazon** [comment]
@koying if necessary, i can bring a debug log

**Comment by ironMann** [comment]
@Rohazon I suspect your frame rate switching is not working also if you run in HDCP mixed mode `|>- 2.2 -> | avr | >- 1.4 -> | TV |`.

Can you try playing a video in passthrough mode (starts without the audio) and then toggling the passthrough option off/on in the Audio settings for the current playback (not the master switch in the System menu). This fixes the passthrough for the current playback for me.

**Comment by chevyguy** [comment]
Pass through stopped working for me as well, after the first video was played. I could play one video, then the next wouldn't work, no sound at all. This is on an Nvidia Shield 2017 and the latest version of SPMC in the App Store on the Shied, using IEC pass through. I am using the Pulse CCM build.

**Comment by Rohazon** [comment]
@ironMann you are right, i'm in "mix mode" and my framerate switching isn't working anymore, sadly toggling passthrough didn't work for me, will spend a few bucks a bit later for a second 2.2 hdmi cable.

**Comment by tknice** [comment]
Same here on a Shield 2015.  Passthrough broke a few weeks ago.  Unfortunately, someone was house sitting and didn't see what was updated.  I'm pretty sure it was the 5.1 OS that broke it.  Doesn't work in SPMC, Kodi, or Plex.

**Comment by grammetry** [comment]
My Shield TV audio-passthrough work again.
I find the reason.
My AVR is Onkyo RZ800 that have 2 HDMI Out.
Yesterday when I switch to sub HDMI Out watch something the audio is abnormal.
Then I go to my AVR setting menu for checking.
I found my Audio TV out (Sub) set On.
I turn it to Off then everything working fine.
The Netflix APP have 5.1 and I use SPMC watch file with DTS-HD MA is also working.
I am so happy, 
Hope you can find the solution, too.
![s__28336133](https://cloud.githubusercontent.com/assets/16472107/26043210/2bed579a-396e-11e7-80d5-f029bc7a6447.jpg)

**Comment by Rohazon** [comment]
I finally got passthrough work again after receiving nvidia 5.2 update,
